application/backend/src/main/resources/apps/typerefinery/components/structure/page/clientlibs/functions.js

```diff
@@ -52,6 +52,9 @@ window.MessageService.Client = MessageService.Client || {};
                             }else if ($component.getAttribute('data-module') === 'linechartComponent') {
                                 componentNs?.Graphs?.LineChart?.dataReceived(payload.data, $component)
                             }
+                            if ($component.getAttribute('data-module') === 'smallTickerComponent') {
+                                componentNs?.Widgets?.Ticker?.dataReceived(payload.data, $component)
```

Comment:
Please test them before PR.

I don't think that for the column ticker you are using the right namespaces.

Reply:
Done